README.md

```diff
@@ -46,67 +44,116 @@
   </a>
 </p>
 
+<br/>
+
+
+### All your data and analytics. Unified. Interactive.
+
 Enso is a general-purpose programming language and environment for interactive
 data processing. It is a tool that spans the entire stack, going from high-level
 visualisation and communication to the nitty-gritty of backend services, all in
 a single language.
 
-- 🔗 **Connect to all the tools you're already using**  
-  Enso ships with a robust set of libraries, allowing you to work with local
-  files, databases, HTTP services and other applications in a seamless fashion.
-  <br/><br/>
-- 📊 **Cutting-edge visualization engine**  
-  Enso is equipped with a highly-tailored WebGL visualization engine capable of
-  displaying even millions of data points 60 frames per second in a web
-  browser.<br/><br/>
-- 🌐 **Polyglot**  
-  Enso allows you to use any Java library in your code. Soon, it will also allow
-  you to copy-paste code from Python, JavaScript, Ruby, and R with close-to-zero
-  performance overhead at runtime.<br/><br/>
-- ⚡ **High performance**  
-  Enso graphs and code can run up to 100x faster than the analoguous Python
-  code.<br/><br/>
-- 🛡️ **Results you can trust**  
-  Enso incorporates many recent innovations in data processing and programming
-  language design to allow you to work quickly and trust the results that you
-  get. It is a purely functional programming language with higher-order
-  functions, user-defined algebraic datatypes, pattern-matching, and a rich set
-  of primitive types.<br/><br/>
-- 🌎 **Runs everywhere**  
-  Enso is available on MacOS, Windows, and Linux, and the Enso IDE runs on
-  web-native technologies. In time, you'll be able to run it in the web-browser,
-  giving even your tablet of phone access to your data.<br/><br/>
-
-<a href="https://www.youtube.com/watch?v=XReCQMZUmuE">See it in action.<br>
-
-<img alt="An example Enso graph" src="https://user-images.githubusercontent.com/1623053/105841783-7c1ed400-5fd5-11eb-8493-7c6a629a84b7.png">
-</a>
+<img align="left" width="36px" src="https://github.com/google/material-design-icons/blob/master/src/content/link/materialiconsround/24px.svg">
```

Comment:
All of these icons are almost impossible to see on dark-mode GH.

Reply:
We already have this issue with the logo on top of the page. The proper solution requires a lot of tricks (we cannot use predefined logos anymore, we need to fork them and change them). I'd propose to design README for now for the white theme and when we will be happy with it (I'm definitely not yet) then we will make it working in the dark mode as well. The reason is also that I'm not even sure if we will keep all the icons, as we will add here some graphics done by our website graphic designer, so lets first finish the README to an amazing state and then lets add dark omde support! :)